index.html

```diff
@@ -0,0 +1,117 @@
+        <nav id="side-nav">
+            <a class="fa fa-bars" href="#"></a>
+            <div class="portfolio">
+                <a href="#">Portfolio</a>
+            </div>
+            <div class="about">
+                <a href="#">About</a>
+            </div>
+            <div class="contact">
+                <a href="#">Contact</a>
+            </div>
+            <div class="mail">
+                <a href="#">
+                    <i class="fa fa-envelope-o"></i>
+                </a>
+            </div>
+
+        </nav>
+
+    </header>
+
+    <section class="header">
+        <h1>Patrick Maina</h1>
+        <h2>Software Developer</h2>
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.
+        </p>
+        <div class="abilities">
+            <ul class="languages">
+                <li>Language</li>
+                <li>Javascript</li>
+                <li>Ruby</li>
+                <li>Html</li>
+                <li>Css</li>
+            </ul>
+
+            <ul class="frameworks">
+                <li>Frameworks</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+                <li>CapyBara</li>
+                <li>Selenium</li>
+                
+            </ul>
+
+            <ul class="skills">
+                <li>Skills</li>
+                <li>Codekit</li>
+                <li>Github</li>
+                <li>Codepen</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+                
+            </ul>
+        
+        </div><hr>
+        <div class="footer">
+            <a class="text" href="#">work
+                <i class="fa fa-arrow-down"></i>
+            </a>
+            <ul>
+                <li>
+                    <a href="#">
+                        <i class="fa fa-github"></i>
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <i class="fa fa-linkedin"></i>
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <i class="fa fa-angellist"></i>
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <i class="fa fa-twitter"></i>
+                    </a>
+                </li>
+
+                <li>
+                    <a href="#">
+                        <i class="fa fa-medium"></i>
+                    </a>
+                </li>
+            </ul>
+            <a class="text" href="#">work
+                <i class="fa fa-arrow-down"></i>
+            </a>
+            
+        </div>
+
+    </section>
+    <script src="https://kit.fontawesome.com/03046ec777.js" crossorigin="anonymous"></script>">
+      
```

Comment:
- [ ] Kindly remove this `work` and place it after the icons as Figma design
### your design
![hhhh](https://user-images.githubusercontent.com/81727988/170519164-80cda070-145f-444c-8103-443d2f82272e.png)
### Figma design

![gggg](https://user-images.githubusercontent.com/81727988/170519455-4cd77567-72e6-440a-898b-6b4388f5e668.png)

Reply:
resolved :heavy_check_mark: